lib/vsc/install/commontest.py

```diff
@@ -193,35 +253,17 @@ def test_importscripts(self):
                                  msg='check_header of %s' % scr)
 
     def test_prospector(self):
-        """Run prospector.run.main, but apply white/blacklists to the results"""
-        orig_expand_default = optparse.HelpFormatter.expand_default
-
-        if not HAS_PROSPECTOR:
-            if sys.version_info < (2, 7):
-                log.info('No protector tests are ran on py26 or older.')
-            else:
-                log.info('No protector tests are ran, install prospector manually first')
-
-                # This is fatal on jenkins/...
-                if 'JENKINS_URL' in os.environ:
-                    self.assertTrue(False, 'prospector must be installed in jenkins environment')
-
-            return
-
-        sys.argv = ['fakename']
-        sys.argv.extend(self.PROSPECTOR_OPTIONS)
-        # add/set REPO_BASE_DIR as positional path
-        sys.argv.append(self.setup.REPO_BASE_DIR)
+        """Run prospector and apply white/blacklists to the results"""
 
-        config = ProspectorConfig()
-        # prospector will sometimes wrongly autodetect django
-        config.libraries = []
-        prospector = Prospector(config)
+        if not HAS_PROSPECTOR and sys.version_info >= (2, 7) and 'JENKINS_URL' in os.environ:
+            # This is fatal on jenkins/...
```

Comment:
is this still an issue? can you retry and see what goes wrong and open an issue?